lib/mosaic-routing/src/test/java/org/eclipse/mosaic/lib/routing/graphhopper/util/DatabaseGraphLoaderTest.java

```diff
@@ -122,66 +117,64 @@ public void correctImport() throws Exception {
         it = expl.setBaseNode(n2);
         assertionMap.put(n1, edgeAssertion(n1, con1_2_1, 80, 140, false));
         assertionMap.put(n3, edgeAssertion(n3, con3_2_3, 50, 140, false));
-        assertNextEdge(enc, it, assertionMap);
-        assertNextEdge(enc, it, assertionMap);
+        assertNextEdge(vehicleEncoding.speed(), wayTypeEnc, it, assertionMap);
+        assertNextEdge(vehicleEncoding.speed(), wayTypeEnc, it, assertionMap);
         assertTrue(assertionMap.isEmpty());
         assertFalse(it.next());
 
         //assert outgoing edges of node 3
         it = expl.setBaseNode(n3);
         assertionMap.put(n2, edgeAssertion(n2, con3_3_2, 50, 140, false));
-        assertNextEdge(enc, it, assertionMap);
+        assertNextEdge(vehicleEncoding.speed(), wayTypeEnc, it, assertionMap);
         assertTrue(assertionMap.isEmpty());
         assertFalse(it.next());
 
         //assert outgoing edges of node 4
         it = expl.setBaseNode(n4);
         assertionMap.put(n2, edgeAssertion(n2, con2_4_2, 50, 30 * 3.6 * 0.9, true));
-        assertNextEdge(enc, it, assertionMap);
+        assertNextEdge(vehicleEncoding.speed(), wayTypeEnc, it, assertionMap);
         assertTrue(assertionMap.isEmpty());
         assertFalse(it.next());
 
-        //assert turn restriction
-        TurnCostEncoder turnEnc = (TurnCostEncoder) enc;
 
         //U-Turns have high costs after the import process (u-turn penalty + turn costs)
-        assertFalse(turnEnc.isTurnRestricted(tcStorage.getTurnCostFlags(con1_2_1, n1, con1_1_2)));
-        assertFalse(turnEnc.isTurnRestricted(tcStorage.getTurnCostFlags(con3_2_3, n3, con3_3_2)));
-        assertEquals(122, turnEnc.getTurnCost(tcStorage.getTurnCostFlags(con1_2_1, n1, con1_1_2)), 0.1d);
-        assertEquals(122, turnEnc.getTurnCost(tcStorage.getTurnCostFlags(con3_2_3, n3, con3_3_2)), 0.1d);
+        assertFalse(tcStorage.get(vehicleEncoding.turnRestriction(), con1_2_1, n1, con1_1_2));
+        assertFalse(tcStorage.get(vehicleEncoding.turnRestriction(), con3_2_3, n3, con3_3_2));
+        assertEquals(130, tcStorage.get(vehicleEncoding.turnCost(), con1_2_1, n1, con1_1_2), 0.1d);
```

Comment:
check

Reply:
I think the value increased since we now add turn-costs AND u-turn-penalty. Before it was u-turn-penalty.